odpf/optimus/runtime_service.proto

```diff
@@ -680,6 +680,8 @@ message ReplayRequest {
     string start_date = 4;
     string end_date = 5;
     bool force = 6;
+    bool ignore_downstream = 7;
+    string allowed_downstream = 8;
 }
 
 message ReplayResponse {
```

Comment:
Let's add ignored_jobs in the replay response too.

Reply:
added ignored_jobs